SubmittingPatches-backports.rst

```diff
@@ -221,7 +221,7 @@ which automates the process and takes away most of the guesswork.
 The ceph-backport.sh script
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-Similar to the case of creating the `Backport tracker issues`_, staging the actual
+Similar to the case of creating the `Creating Backport tracker issues`_, staging the actual
```

Comment:
please remove the two words "creating the" before the hyperlink (as it is now, it reads "creating the Creating")

Reply:
done